base/array.jl

```diff
@@ -2323,8 +2463,7 @@ function findall(pred::Fix2{typeof(in),<:Union{Array{<:Real},Real}}, x::Array{<:
 end
 # issorted fails for some element types so the method above has to be restricted
 # to element with isless/< defined.
-findall(pred::Fix2{typeof(in)}, x::AbstractArray) = _findin(x, pred.x)
-findall(pred::Fix2{typeof(in)}, x::Tuple) = _findin(x, pred.x)
+findall(pred::Fix2{typeof(in)}, x::Union{AbstractArray, Tuple}) = _findin(x, pred.x)
 
```

Comment:
Looks like something went wrong here

Reply:
Ah, maybe something got confused in a merge.